Makefile

```diff
@@ -53,12 +53,12 @@ format: ## Format Go source files
 test: test-unit
 
 .PHONY: test-unit
-test-unit: download-tokenizer
+test-unit: download-tokenizer install-zmq
```

Comment:
nit: is the distinction between verbs (`download` for tokenizer, `install` for zmq) needed? Should try to use consistent makefile targets, if possible

Reply:
`download-tokenizer` downloads the tokenizer lib to `/lib` while `install-zmq` uses a package manager to "install" the library. Technically both do download to some path, but the latter does more. Do you think `install` fits both?

Reply:
That is true and was what I initially wanted to go for - but I could not find any formal repository that conveniently provides the builds per arch, therefore followed the official guide. Although looking at it now, they called it "download" https://zeromq.org/download/.